packages/liquidator/src/liquidationStrategy.js

```diff
@@ -66,12 +61,11 @@ module.exports = (
   assert(BN, "requires BN");
 
   // these variables are optional and are only checked if WDF is activated
-  whaleDefenseFundWei &&
-    assert(
-      defenseActivationPercent >= 0 && defenseActivationPercent <= 100,
-      "Requires defenseActivationPercent to bet set between 0 and 100"
-    );
-  whaleDefenseFundWei && assert(withdrawLiveness > 0, "requires withdrawLiveness");
+  const wdfActive = !isNaN(defenseActivationPercent) && defenseActivationPercent > 0;
```

Comment:
Previously, setting `whaleDefenseFundWei > 0` would activate the WDF. Now, setting `defenseActivationPercent >0` activates it.